react-components/src/hooks/network/getAssetsList.ts

```diff
@@ -0,0 +1,44 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+import { type CogniteClient, type CursorResponse, type Asset } from '@cognite/sdk';
+import { buildFilter } from '../../utilities/buildFilter';
+
+export const getAssetsList = async (
+  sdk: CogniteClient,
+  {
+    query,
+    cursor,
+    limit = 1000,
+    sort = [{ property: ['_score_'] }],
+    aggregatedProperties = ['path']
+  }: {
+    query: string;
+    cursor?: string;
+    limit?: number;
+    sort?: Array<{ property: string[] }>;
+    aggregatedProperties?: string[];
+  }
+): Promise<{ items: Asset[]; nextCursor: string | undefined }> => {
+  const advancedFilter = buildFilter(query);
+
+  return await sdk
+    .post<CursorResponse<Asset[]>>(`/api/v1/projects/${sdk.project}/assets/list`, {
```

Comment:
Can we use the sdk version of it?
`sdk.assets.list()`

Reply:
we are using advanced filter which I could not find the support with sdk.assets.list()

Reply:
Oh yeah.. right